api/app/Models/User.php

```diff
@@ -119,7 +119,7 @@ public function expectedGenericJobTitles(): BelongsToMany
 
     public function isAdmin(): bool
```

Comment:
Policies should probably be checking permissions, not this function.  Removing this is out of scope for this PR, I think, but could you mark it as deprecated?

Reply:
See: f48ed22243cd874cdc5316afde55cea04620e5fc